CHANGELOG.md

```diff
@@ -56,3 +56,10 @@ All notable changes to the "discord-py-snippets" extension will be documented in
 - Added `!cooldown` decorator
 - Added `!waitforreaction` snippet
 - Added `!waitformessage` snippet
+
+
+### [1.4.1]
```

Comment:
```suggestion
### [1.5.0]
```

I would rather want this to be named version 1.5.0 since this project follows [semantic versioning](https://semver.org) which states

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
> 1. MAJOR version when you make incompatible API changes,
> 2. MINOR version when you add functionality in a backwards compatible manner, and
> 3. PATCH version when you make backwards compatible bug fixes.

Since this adds some functionality and is not really just a bug fix, 1.5.0 is better